docs/developers/basics/creating-an-addon.md

```diff
@@ -45,6 +45,58 @@ See [this source](https://wiki.facepunch.com/gmod/Lua_Folder_Structure) for a de
 
 ## Publishing your Addon
 
+## Windows
+
+In order to upload your addon to the Steam Workshop, you need to first compile it into a .gma file.
+
+1. Your addon file must contain an "addon.json" within the `<addon name>/` directory of your folder. This consists of a key:value pair table with information used by the Steam
+Workshop to create tags for uploads.
+    It should have a structure like this:
+
+    ![addon_json.png](../../assets/images/article/addon_json.png)
```

Comment:
I'd recommend adding `???+ abstract "TargetID Usage Examples"` in the line before so the image is placed inside a box.

See: https://raw.githubusercontent.com/TTT-2/docs/master/docs/developers/content-creation/using-targetid.md

Reply:
ohhhhhh I wondered how you got the fancy boxes, I had trouble reading up on MKDocs

Reply:
Here's the reference: https://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
thank you